README.md

```diff
@@ -3,28 +3,29 @@
 TO COMPILE PROJECTIONS:
```

Comment:
Would suggest changing the case in this heading to title case or so. Also, please change to markdown headings, i.e. `# Compiling Projections`.